amcl/src/amcl_node.cpp

```diff
@@ -1045,6 +1096,93 @@ AmclNode::getOdomPose(geometry_msgs::PoseStamped& odom_pose,
   return true;
 }
 
+pf_vector_t
+AmclNode::getOdomMovement(const pf_vector_t& now_pose, const ros::Time& now,
+                              const ros::Time& get_odom_time)
+{
+  // To match the frame of amcl_base_link
+  geometry_msgs::TransformStamped tf_now, tf_past;
+  pf_vector_t move = pf_vector_zero();
+  try
+  {
+    tf_now = tf_->lookupTransform(global_frame_id_, base_frame_id_, now);
+  }
+  catch (tf2::TransformException& ex)
+  {
+    ROS_WARN("Could not get now odom tf: %s", ex.what());
```

Comment:
Could not get the current odom tf

Reply:
done in b6de6ed965ac355d831b018847ab2800f4bb51e4